simple/src/main/java/org/apache/commons/rdf/simple/experimental/AbstractRDFParser.java

```diff
@@ -200,7 +200,7 @@
 	}
 
 
-	private Optional<RDFTermFactory> rdfTermFactory = Optional.empty();
```

Comment:
Yes, I think we should review this post 0.3.0 as we de-experimentalize `AbstractRDFParser` and make equivalent `Writer` - there's nothing else preventing it from being serializable - the `RDF` factory itself must also be serializable though.

Raised as
[COMMONSRDF-49](https://issues.apache.org/jira/browse/COMMONSRDF-49)